src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
```

Comment:
Better to use `if` because it will exist only when you need. Now you will always do `switch`, which could play with you bad joke in larger project and also it's overload memory.